script/CartController.js

```diff
@@ -0,0 +1,134 @@
+import createElement from "./CreateElement.js";
+
+export function initCart(){
+    const bordered_box = document.querySelector('.border_box');
+    const box_inner = document.querySelector('.border_inner');
+    const cart_inner = document.querySelector('.cart_inner');
+
+    let currentCard = '';
+    let totalCart = 0;
+    let arr = [];
+    const cards = document.querySelectorAll('.card');
+    bordered_box.addEventListener('dragover', dragOver);
+    bordered_box.addEventListener('drop', dragDrop);
+    let i = 1;
+    cards.forEach(function (element) {
+        element.id = i;
+        i++;
+        element.addEventListener('dragstart', dragStart);
+        element.querySelector('.button').addEventListener('click', function () {
+            currentCard = document.getElementById(this.closest('.card').id);
+            let cloneCurrentCard = currentCard.cloneNode(true);
+            addToCart(cloneCurrentCard)
+        });
+    });
+
+    function dragStart() {
+        if (!bordered_box.classList.contains('active')) {
+            bordered_box.className += ' active';
+        }
+        currentCard = this;
+    }
+
+    function dragOver(e) {
+        e.preventDefault();
+    }
+
+    function dragDrop() {
+        let cloneCurrentCard = currentCard.cloneNode(true);
+        addToCart(cloneCurrentCard);
+    }
+
+    function cartTemplate(img, title, author, price, id) {
```

Comment:
When function requires a lot of arguments, we can use object approach, so we can pass arguments in any order
```suggestion
    function cartTemplate({ img, title, author, price, id }) {
```